deepchem/data/datasets.py

```diff
@@ -1371,7 +1371,7 @@ def move(self, new_data_dir: str,
       before moving. This is set to True by default to be backwards compatible
       with behavior in earlier versions of DeepChem.
 
-    Notes
+    Note
     -----
```

Comment:
I think you have an extra `_` here and in a couple places below

Reply:
I fixed!